runbotics/runbotics-orchestrator-ui/src/src-app/components/InfoPanel/IterationSlide/IterationSlide.tsx

```diff
@@ -84,10 +84,8 @@ const IterationSlide: FC<IterationSlideProps> = ({
                                     {iterationGutter.iteratorElement && (
                                         <ReactJson
                                             src={{
-                                                iterator: JSON.parse(
-                                                    iterationGutter.iteratorElement ??
-                                                        '{}'
-                                                ),
+                                                iterator: 
+                                                    JSON.parse(iterationGutter.iteratorElement as string)
```

Comment:
are u sure? here can be json also, `JSON.parse` wont throw an error? you can create a fn that will cover each possible type

Reply:
im pretty sure

Reply:
JSON parse is a wierd thing, but it works with "1", "\"n\"", "{}", and thats our options